languages/ruby/exercises/concept/floating-point-numbers/.meta/example.rb

```diff
@@ -0,0 +1,41 @@
+module SavingsAccount
+
+  NEGATIVE_RATE = -3.213
+  SMALL_POSITIVE_RATE = 0.5
+  MEDIUM_POSITIVE_RATE = 1.621
+  LARGE_POSITIVE_RATE = 2.475
```

Comment:
Should these constants be in the [scaffold](https://github.com/exercism/v3/pull/1168/files#diff-60c0b47ba0741412e493e7e14c68945c) ?

Reply:
No, I don't think so. Let people define them, as we've already talk constants in `basics`. And if they don't, that's something the analyzer can push people towards to avoid "magic numbers"